Source/Radiation/SpectralModels.H

```diff
@@ -58,6 +58,11 @@ getRadPropGas(
   amrex::GpuArray<amrex::Real, 126UL> const& kdatah2o,
   amrex::GpuArray<amrex::Real, 126UL> const& kdataco)
 {
+  if (
+    yco2(i, j, k) < 1.0e-5 && yh2o(i, j, k) < 1.0e-5 && yco(i, j, k) < 1.0e-5) {
+    absc(i, j, k) = 0.001;
+    return;
+  }
```

Comment:
Just want to make sure I get what's going on. Shouldn't this not be needed because later on this variable is limited?  https://github.com/AMReX-Combustion/PelePhysics/blob/80f2cfa0edd2e928a93bb9dff45e68d32d8699f7/Source/Radiation/PeleCRad.H#L125

Reply:
The guard you showed failed in the EB production case (PeleLMeX case not PeleC) I've been testing. 
And the additional guard in this PR fixed the issue for some reason, guessing some EB cell might be the cause.

It might only need one guard, but now I didn't have a chance to test all my previous cases, will do that when possible.

Reply:
That's weird, but I guess it shouldn't hurt to have an extra guard. Although it might be a sign of some other unknown bug.